code-challenges.test.js

```diff
@@ -14,39 +14,78 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
 
+
+
+
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
+// describe("isDivisible", () => {
+//     it("a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+
+//     const object1 = { number: 15 } 
+
+//     const object2 = { number: 0 }
+
+//     const object3 = { number: -7 }
+//       expect(isDivisble(object1)).toEqual("15 is divisible by three")// Expected output:
+//       expect(isDivisble(object2)).toEqual("0 is divisible by three")// Expected output:
+//       expect(isDivisble(object3)).toEqual("-7 is not divisible by three")// Expected output:
+//     })
+//   })
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
 
-// Pseudo code:
 
+// // Pseudo code: input: an object that passes through a function and checks if the number value is divisble by 3
+//                 // output: return a string that says "${object.number} is/is not divisble by 3"
+//                 //tenary operater to evalue object.number and return desired string
+
```

Comment:
Remember that pseudo code is not just rewriting the prompt. It is your place to conceptually walk through what your code is doing.